docs/zh-CN/docs/sql-manual/sql-reference/Data-Definition-Statements/Alter/ALTER-TABLE-PARTITION.md

```diff
@@ -64,6 +64,8 @@ partition_desc ["key"="value"]
 - 如指定分桶方式，只能修改分桶数，不可修改分桶方式或分桶列。如果指定了分桶方式，但是没有指定分桶数，则分桶数会使用默认值10，不会使用建表时指定的分桶数。如果要指定分桶数，则必须指定分桶方式。
 - ["key"="value"] 部分可以设置分区的一些属性，具体说明见 [CREATE TABLE](../Create/CREATE-TABLE)
 - 如果建表时用户未显式创建Partition,则不支持通过ALTER的方式增加分区
+- 如果用户使用的是List Partition则可以增加default partition
+  -  ALTER TABLE ADD PARTITION DEFAULT
```

Comment:
Please explain what is default partition, eg, which value will be saved in default partition